nautobot/core/settings.py

```diff
@@ -249,27 +249,6 @@
     }
 }
 
-#
-# Redis (Caching/Queuing)
-#
-
-REDIS = {
-    "tasks": {
-        "HOST": "localhost",
-        "PORT": 6379,
-        "PASSWORD": "",
-        "DATABASE": 0,
-        "SSL": False,
-    },
-    "caching": {
-        "HOST": "localhost",
-        "PORT": 6379,
-        "PASSWORD": "",
-        "DATABASE": 1,
-        "SSL": False,
-    },
-}
-
```

Comment:
Intentional to remove this? If so, why?

Reply:
Oh sorry I put that into the commit message. That was a legacy setting that we replaced when refactoring "magic" settings from NetBox to "explicit" settings in Nautobot, and it was still lingering there in the core settings.